### PR TITLE
Vulcan: Add breadcrumbs to page

### PR DESCRIPTION
**Pull request by andyg0808**
This adds the breadcrumbs to the page. It's using the existing API we added in https://github.com/iFixit/ifixit/pull/46347.

## QA Notes
Please check that the breadcrumbs seem reasonable and look as-expected.

Please also check that the page looks reasonably like the mockup at a mobile size.

Closes https://github.com/iFixit/ifixit/issues/46770
Connects https://github.com/iFixit/ifixit/issues/46879

### CODE REVIEW

**Review by jyee27 — APPROVED** [review]
CR ☕ through [98fdb84](https://github.com/iFixit/react-commerce/pull/1439/commits/98fdb84901e67ae5c861b6bbf69c70263fdfba4e)

**Comment by jyee27** [comment]
CR ☕ through [a3f928b](https://github.com/iFixit/react-commerce/pull/1439/commits/a3f928bee98b4c83e966f9e3c1c59a8024cc3437)

**Review by jyee27 — APPROVED** [review]

**Comment by deltuh-vee** [comment]
I think the breadcrumbs should be a little more aligned with the rest of the content.
image

**Review by hcutrone — DISMISSED** [review]
CR :burrito:

**Comment by andyg0808** [comment]
un_dev_block :+1: 
Breadcrumbs should be aligned, and tests should pass.

**Comment by erinemay** [comment]
dev_block 🧷 
I'm seeing a scrollbar on the Parts/Guides/Answers block now. This is Chrome at 100% zoom. 
image
https://react-commerce-git-vulcan-add-breadcrumbs-ifixit.vercel.app/vulcan/troubleshooting-test

And some merge conflicts.

Otherwise, the breadcrumbs look good
https://react-commerce-git-vulcan-add-breadcrumbs-ifixit.vercel.app/vulcan/erins_troubleshooting_test
image

deploy_block ❔ We don't show the guide page in breadcrumbs on the new guides. I'm wondering if we should do the same here and show the last breadcrumb as the device page. 
https://www.cominor.com/Guide/MacBook+Pro+13-Inch+Unibody+Mid+2010+AirPort+Antenna+Replacement/6025
image

**Comment by erinemay** [comment]
Shoot, @andyg0808, I totally missed this 🤦. dev_block 😢 
The letters are getting cut off on the bottom. You can see it the clearest on the g in my title. This is Windows + Chrome.
https://react-commerce-git-vulcan-add-breadcrumbs-ifixit.vercel.app/vulcan/erins_troubleshooting_test
image

Also, after talking to @sctice-ifixit and looking at the mockup, we decided to end the breadcrumbs at the device (just like on guide pages).
[Guide](https://www.cominor.com/Guide/MacBook+Pro+13-Inch+Unibody+Mid+2010+AirPort+Antenna+Replacement/6025):
image

[Here](https://react-commerce-git-vulcan-add-breadcrumbs-ifixit.vercel.app/vulcan/erins_troubleshooting_test):
image

**Comment by andyg0808** [comment]
un_dev_block :+1: 
I think I fixed the letters getting cut off, and I think the breadcrumbs should look good on small screens now.

**Comment by jordycosta** [comment]
@andyg0808 currently running into a 500 Internal Server Error

**Comment by andyg0808** [comment]
@jordycosta Should be fixed now. The newish Cominor DB didn't have the `troubleshooting-test` wiki; I copy-pasted the source text back into a new version.

**Comment by deltuh-vee** [comment]
dev_block ✊ 
I'm seeing scrollbars again.

CleanShot 2023-03-31 at 15 00 09

**Comment by andyg0808** [comment]
un_dev_block :+1: 
As far as I know, that's correct and expected. I decided to use scrollbars here when there's not room enough to show all the tabs (note how the `Answers` tab is scrolled off the screen here). It does look like I'm not getting the fade I was intending to, however.

**Comment by sctice-ifixit** [comment]
> I decided to use scrollbars here when there's not room enough to show all the tabs

But why isn't there enough room? Shouldn't the breadcrumbs be collapsing into the menu before we get to this point?

**Comment by k0rvusk0r4x** [comment]
Breadcrumbs look good except the scroll bar, which has been addressed.

QA 😺 

deploy_block 😿 to respond to [this](https://github.com/iFixit/react-commerce/pull/1439#issuecomment-1494515548) and fix or perhaps make a follow-up issue.

**Comment by andyg0808** [comment]
@sctice-ifixit That's a great question, and I think I've got more layers of flexbox involved than I'm able to mentally compute against. I've taken @k0rvusk0r4x's advice and opened [#47235](https://github.com/iFixit/ifixit/issues/47235) to figure that out, since it'd be really nice to stop having this pull in limbo.

File: packages/breadcrumbs/Breadcrumbs.tsx [hunk 1]
```diff
@@ -142,7 +142,7 @@ const IFixitBreadcrumb = memo(function IFixitBreadcrumb({
             ...hiddenWrapProps,
             flexDirection: 'row-reverse',
             justifyContent: 'flex-end',
-            height: 'min-content',
+            height: '1.5em',
```
Comment (jyee27) [inline packages/breadcrumbs/Breadcrumbs.tsx:145]: We should check that this works on languages with non-Latin characters.

**Review by jyee27 — APPROVED** [review]
CR ☕

**Comment by jyee27** [comment]
deploy_block ☕ on https://github.com/iFixit/react-commerce/pull/1439#issuecomment-1497024459

**Comment by erinemay** [comment]
Good callout @jyee27.

image
image
image

It looks ok with KO, ZH, RU, JP, TR, and PT characters. 
https://react-commerce-git-vulcan-add-breadcrumbs-ifixit.vercel.app/vulcan/erins_troubleshooting_test
https://www.cominor.com/Device/PDA

un_deploy_block 🧷

**Comment by erinemay** [comment]
But actually maybe deploy_block 🦺 on @andyg0808 or @sctice-ifixit responding to https://github.com/iFixit/react-commerce/pull/1439#issuecomment-1497024459

**Comment by andyg0808** [comment]
@erinemay Thanks; I think @dhmacs has a point, and I probably should try those breadcrumbs before we ship this.
dev_block :+1: on that.

**Comment by dhmacs** [comment]
> I probably should try those breadcrumbs before we ship this

@andyg0808 feel free to copy paste the code of the navbar and adapt it to this use case, we can create a reusable component in another PR (current product list breadcrumbs implementation still receives a product list as a prop, but eventually I'd like to create a component that is agnostic a just receives an array of breadcrumbs)

**Comment by andyg0808** [comment]
Talked to @sctice-ifixit, and I've updated https://github.com/iFixit/ifixit/issues/47235 to recommend using the product list page breadcrumbs.

un_dev_block :+1: so that we can just ship this.

**Pull request merged**